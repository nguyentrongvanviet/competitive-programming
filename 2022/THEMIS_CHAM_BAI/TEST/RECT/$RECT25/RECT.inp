181
-453561678 -569677061
527805225 132517332
-587319958 -320518005
609150818 563148866
-715047228 302535002
-862584982 513066753
166136371 711819209
-846426165 58176399
-696057958 198810055
-600391707 239892029
-464922173 -609079311
-651135849 289372755
401897918 -642913939
-899746737 349557778
-46139534 310380098
34872364 -4900087
468579345 213860684
-962827305 424499050
-432596641 659804031
-587319958 -320518005
201838130 45445637
-101318169 645046392
-244935677 -77054634
881971666 878831349
-656605096 373840567
-173227281 68071580
-282720243 985264959
-30962437 622484872
521782070 -872921537
76872990 689300688
-738324758 -259341507
-918004409 -799305647
302830646 -58757880
166136371 711819209
-284541181 429317530
-385594970 955209627
-428882858 666607975
-673878936 55783964
585252193 -430542207
-611485318 995830003
-367095694 882271419
-560620391 869717206
888989910 236386882
-899746737 349557778
10729235 791154901
506838749 -40950940
208414017 -928101778
-596887799 912670623
966414202 -922592213
150091337 91736150
-273638308 -210710638
-313470584 146998458
-188151313 -797232457
-432596641 659804031
767880156 256568396
-464922173 -609079311
71741320 853624827
-490618498 -232824294
-738324758 -259341507
-452855580 -930655451
-989205278 -426173253
76274441 646994484
15205315 -227185579
170819492 -472443998
-907257923 47858256
-854775175 -796839572
860829799 -265708540
91529658 -205361844
-313470584 146998458
-870186586 -885131958
208414017 -928101778
622292502 438786433
432043974 -995295556
76872990 689300688
-949886314 517743281
609150818 563148866
182487797 -603068867
340231236 -238370766
723373615 797261273
-71734517 83578539
738402440 588201781
-31645765 -184333631
-30962437 622484872
-765463191 248131707
975512184 704412214
-840384671 -407468487
401758877 -412916410
507635471 -163909899
429343790 484695959
546198970 -3201399
527805225 132517332
546776017 798031954
-215984560 662127931
705683449 590140035
750672443 288201070
-656605096 373840567
376942442 -487625060
-558730990 -330617073
416334213 503267169
323290532 -634084148
-910711745 -941090286
873325030 173115690
-810351135 378315531
-408748408 -461452469
-355685557 820725142
511164807 -268706724
-602687034 -285959295
-428882858 666607975
-308619958 501451172
338542271 -357387865
335862239 -358736286
-949886314 517743281
-256941191 -686469762
-82727824 -592547182
-611485318 995830003
-753042390 5938755
-86487185 621021363
507635471 -163909899
-244935677 -77054634
-91136407 -92978665
92706095 357050079
150091337 91736150
-114340998 965383475
-802147952 322907111
-918292804 991687778
985659600 -67713509
773478504 -767235151
558200942 847941876
72677874 859739659
229246188 -855291349
773771198 -451974096
773332222 92279590
-592408720 736695273
379384766 -493349188
-90606763 607675446
482061852 453110766
11392056 698001168
-602687034 -285959295
-918004409 -799305647
-173227281 68071580
-945132027 -445765019
-91136407 -92978665
716537416 -540415742
-753042390 5938755
-408748408 -461452469
-273638308 -210710638
-601903057 719382661
985659600 -67713509
705683449 590140035
482061852 453110766
-507255127 -741906723
939095713 129156399
826430406 -953920464
-61844784 -449694425
455957840 -503372910
338542271 -357387865
472758745 -559714574
860829799 -265708540
-284541181 429317530
520622192 885720620
-552965323 791128608
704054157 -773241028
-514234178 -725368451
-139707520 -603512042
-355685557 820725142
-840557194 -87571776
69521563 255964467
-213405322 282825351
288789941 -86885674
-971758362 615988807
898185813 -308751450
-598066100 -952105762
-256941191 -686469762
34872364 -4900087
-592408720 736695273
622292502 438786433
-889470284 729359718
-287020214 179500169
873325030 173115690
-881548837 -591367324
-714186293 771381052
