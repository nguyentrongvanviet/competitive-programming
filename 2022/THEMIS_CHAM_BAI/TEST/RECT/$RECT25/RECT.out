1095527620704767653